### DISCUSSION TITLE
419 ERROR

### DISCUSSION THREAD

**Discussion by ssimanga** [Laravel-Backpack/community-forum#550] [Q&A (Help)] [answered]
<img width="861" alt="Screenshot 2023-07-04 142834" src="https://github.com/Laravel-Backpack/community-forum/assets/22166547/ba75e7ae-75e4-469a-9510-a3afa71934c7">

After installing backpack v6, getting 419 | page expired error, I can't go to any other page with getting the page expired error

**Comment by pxpm** [comment, marked as answer]
Duplicate of #552
